project/TestWithCoveragePlugin.scala

```diff
@@ -1,140 +0,0 @@
-import java.time.{ ZonedDateTime, ZoneId }
-import java.time.format.DateTimeFormatter
-import sbt._
-import sbt.Keys._
-import sbt.plugins.JvmPlugin
-import scala.util.Properties
-import scoverage.{Coverage, IOUtils, Serializer}
-import scoverage.report.{CoberturaXmlWriter, ScoverageHtmlWriter, ScoverageXmlWriter}
-
-// Adapted from SCovereageSbtPlugin so it can be run after test even if there are failures.
```

Comment:
Why did we fork the plugin for this? At a minimum, if we decide code coverage is worth it, we should get rid of this custom plugin.